.circleci/devcluster/react.devcluster.yaml

```diff
@@ -0,0 +1,50 @@
+temp_dir: /tmp/priority_scheduler
+
+stages:
+  - db:
+      name: db-priority-scheduler
+      db_name: db-priority-scheduler
```

Comment:
I suspect this was copy-pasted from priority scheduler config, but you are not using the priority scheduler here, so maybe this can be renamed into plain `db`.